uobtheatre/bookings/schema.py

```diff
@@ -35,25 +45,51 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class DiscountRequirementNode(DjangoObjectType):
+    class Meta:
+        model = DiscountRequirement
+        interfaces = (relay.Node,)
+
+
+class DiscountNode(DjangoObjectType):
+    requirements = DjangoListField(DiscountRequirementNode)
+
+    class Meta:
+        model = Discount
+        interfaces = (relay.Node,)
+
+
+class DiscountNodeMixin:
+    discounts = DjangoListField(DiscountNode)
+
+    def resolve_discounts(self, info):
+        return [
+            discount
+            for discount in self.discounts.all()
+            if not discount.is_single_discount()
+        ]
```

Comment:
There are a few options for this which I think might be a bit betterer. One option is to set the discount node queryset but I think the ideal is to somehow set the "default" filtering.

Reply:
james this fix is beyone my level, sooo its all yours bby